src/hooks/series/useEpisode.ts

```diff
@@ -0,0 +1,37 @@
+import { UseQueryResult, useQuery } from 'react-query';
+
+import useMedia from '../useMedia';
+
+import { getSeriesByMediaIds } from '#src/services/api.service';
+import type { ApiError } from '#src/utils/api';
+import type { PlaylistItem } from '#types/playlist';
+import type { Series } from '#types/series';
+
+// Get an episode by id and enrich it with `episodeNumber` and `seasonNumber` for the new series flow
+export const useEpisode = (episodeId: string | undefined, series: Series | undefined): { isLoading: boolean; data: PlaylistItem | undefined } => {
```

Comment:
I don't think useEpisode needs to fetch or return the media item (useMedia.) It's used in SeriesRedirect where we can actually just call useMedia instead and in MediaSeriesEpisode where we already have the episode media item and it would actually be simpler if we just returned the season number and episode number and called it `episodeMetadata` or something to avoid the confusion between `episode` and `enrichedEpisode`

Reply:
Fixed it.